src/main/external-resources/renderers/DefaultRenderer.conf

```diff
@@ -363,6 +363,11 @@ Thumbnails =
 # Default: false
 ThumbnailPadding = 
 
+# The resolution in pixels of thumbnail images.
+# Default: 320 and 180.
+ThumbnailWidth = 
```

Comment:
Another conflict

Reply:
@acanive all that conflicts seem like wrong merging.

Reply:
yes, is wrong merging. fixed in next commit